src/keeshare/ShareExport.cpp

```diff
@@ -29,6 +29,13 @@
 #include <botan/pubkey.h>
 #include <zip.h>
 
+// Compatibility with minizip-ng
+#ifdef MZ_VERSION_BUILD
+#undef Z_BEST_COMPRESSION
+#define Z_BEST_COMPRESSION MZ_COMPRESS_LEVEL_BEST
+#define zipOpenNewFileInZip64 zipOpenNewFileInZip_64
+#endif
+

```

Comment:
I think you should also change the include path to not obscure the `minizip/` prefix. Most of the headers have very generic names and some also exist in zlib iirc. Not including the directory name is asking for this to break again sooner or later.

Reply:
Agreed will include that

Reply:
Change has been made